ros/src/simulation/gazebo_simulator/launcher/vehicle_gazebo_simulation_launcher/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(vehicle_gazebo_simulation_launcher)
```

Comment:
I wanted to align the granularity of stratification. And since it is gazebo_simulator including world, vehicle, launcher, I did not want to make it big package name

Reply:
But I think `{camera, imu, vehicle}_description` package names are too generic and may cause conflicts with other packages in the future. Plz add `gazebo_` prefix or merge to 1 package, e.g. `gazebo_model`?

Reply:
https://github.com/CPFL/Autoware/pull/1930/commits/1e30f34f4754bfb5f4d1740a9e160fc879a483d9